periflow/cli/main.py

```diff
@@ -32,6 +32,7 @@
     context_settings={"help_option_names": ["-h", "--help"]},
     add_completion=False,
     callback=validate_cli_version,
+    pretty_exceptions_enable=False,
```

Comment:
Why did you disable `pretty exceptions`?

Reply:
The pretty exception is too verbose and it displays too much info to users. By applying this change, it will show the normal python traceback. Refer to https://typer.tiangolo.com/tutorial/exceptions/#disable-pretty-exceptions.